apps/utf8.manifest

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
I copied this file from https://learn.microsoft.com/en-us/windows/apps/design/globalizing/use-utf8-code-page#examples. The original file has this line:

```
  <assemblyIdentity type="win32" name="..." version="6.0.0.0"/>
```

I noticed that the name "..." and version "6.0.0.0" in this line look like placeholders. I think we are supposed to replace them with the name and version of avifdec, avifenc, and are_images_equal.

According to Microsoft documentation, the `assemblyIdentity` element is required in an application manifest:
https://learn.microsoft.com/en-us/windows/win32/sbscs/application-manifests

However, I deleted this line and our tests still pass. So I suggest we try going without the `assemblyIdentity` element.

Reply:
Thx for clarifying